core/src/providers/mistral.rs

```diff
@@ -723,6 +753,11 @@ impl MistralAILLM {
             body["tools"] = json!(tools);
         }
 
+        // println!(
```

Comment:
nit

Reply:
They are very convenient in favor of keeping them

Reply:
Quite bearish on commented-out debugging code personally, makes the code look messy and unclean, but as you prefer !

Reply:
kk will remove